src/presentation/ui/component/mappings/mappingsInfos.templ

```diff
@@ -0,0 +1,42 @@
+package componentMappings
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+)
+
+templ MappingsInfos(
+	mapping entity.Mapping, rootDirectory valueObject.UnixFilePath, onClick string,
+) {
+	<div class="bg-os-500 p-4 flex-col rounded mb-1.5">
+		<div class="flex justify-between items-center">
+			<div class="flex space-x-3">
+				<span>{ mapping.Path.String() }</span>
+				if mapping.TargetType.String() == "service" {
```

Comment:
Why is that not a switch? You're evaluating TargetType.String() a lot with those multiples ifs. It's quite hard to read.
https://templ.guide/syntax-and-usage/switch

Reply:
Done!